shoop/xtheme/static_src/editor/style.less

```diff
@@ -169,11 +169,12 @@ form {
     }
 }
 
+@import "../vendor/bower_components/bootstrap/less/variables.less";
+@import "../../../admin/static_src/base/less/variables.less";
```

Comment:
You shouldn't import any variables from admin. Also the vendor variables should be imported inside xhtheme variables.